[dev][compile] failed

**Datart Version**
最新dev
**Describe the bug**
Failed to compile.

./node_modules/@ant-design/pro-table/es/components/ListToolBar/index.less

    &:hover {
      color: @primary-color-hover;
           ^
Variable @primary-color-hover is undefined
      Error in /Users/zy/Downloads/datart-dev/frontend/node_modules/@ant-design/pro-table/es/components/ListToolBar/index.less (line 46, column 13)


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! datart@1.0.0 build: `cross-env GENERATE_SOURCEMAP=false craco build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the datart@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/zy/.npm/_logs/2021-12-23T11_24_27_481Z-debug.log
[ERROR] Command execution failed.
**To Reproduce**
Steps to reproduce the behavior:
1. cd ./datart-dev
2. mvn package
**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environmental details: (optional, depending on where the bug occurs)**
 - OS: mac
 - Source Database: mysql5.7
 - Browser: chrome
 --node v14.18.2
 --npm 6.14.15
**Additional context**
Add any other context about the problem here.

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -83,7 +83,7 @@
   },
   "dependencies": {
     "@ant-design/icons": "^4.5.0",
-    "@ant-design/pro-table": "^2.54.4",
+    "@ant-design/pro-table": "2.60.1",
     "@antv/s2": "^1.3.0",
     "@antv/s2-react": "^1.3.0",
     "@dinero.js/currencies": "^2.0.0-alpha.8",

